stdlib/LinearAlgebra/src/symmetriceigen.jl

```diff
@@ -294,3 +294,36 @@ function eigvals!(A::StridedMatrix{T}, F::LU{T,<:StridedMatrix}; sortby::Union{F
     rdiv!(A, U)
     return eigvals!(A; sortby)
 end
+
+
+function eigen(A::Hermitian{Complex{T}, <:Tridiagonal}; kwargs...) where {T}
+    (; dl, d, du) = parent(A)
+    N = length(d)
+    if N <= 1
+        eigen(parent(A); kwargs...)
+    else
+        if A.uplo == 'U'
+            E = du'
+            Er = abs.(du)
+        else
+            E = dl
+            Er = abs.(E)
+        end
+        S = Vector{Complex{T}}(undef, N)
+        S[1] = 1
+        S[2] = Er[1] == 0 ? one(T) : S[1] * E[1]/Er[1]
```

Comment:
`E[i] / Er[i]` can be replaced by `sign(E[i])` resp.  `conj(sign(du[i]))` resp. `sign(dl[i])`.

Reply:
Nice thanks I've switched to `sign`.